agent/src/main.rs

```diff
@@ -139,6 +139,8 @@ async fn main() -> Result<()> {
         Err(_) => None,
     };
 
+    let host = std::env::var("HOST").unwrap_or_else(|_| "0.0.0.0".to_string());
```

Comment:
Now that we have `MESH_PORT`, Should we rename `HOST` and `PORT` to `HTTP_HOST` and `HTTP_PORT`?